Userland/Libraries/LibGUI/AbstractScrollableWidget.h

```diff
@@ -126,7 +126,7 @@ class AbstractScrollableWidget : public Frame {
 
     RefPtr<Core::Timer> m_automatic_scrolling_timer;
     bool m_active_scrolling_enabled { false };
-    int m_autoscroll_threshold { 20 };
+    int m_autoscroll_threshold { 70 };
```

Comment:
Please remove this commit and the subsequent revert commit from your PR; you only need the last commit